SimplePolicy/SimplePolicyPlugin.cs

```diff
@@ -63,6 +63,7 @@ private static IEnumerable<Transaction> FilterForBlock_Policy2(IEnumerable<Trans
             Transaction[] free = tx_list.Where(p => p.IsLowPriority)
                 .OrderByDescending(p => p.NetworkFee / p.Size)
                 .ThenByDescending(p => p.NetworkFee)
+                .ThenByDescending(p => InHighPriorityList(p))
```

Comment:
I think NetworkFee / Size will always put Claim as last, because Fee is zero. Claim will only get "priority" if other low priority is zero (if everyone sends 0.0000001) no one will ever be able to claim.

Do you think we could change logic to (for free tx):

.OrderByDescending(p => InHighPriorityList(p)) .ThenByDescending(p => p.NetworkFee / p.Size)
.ThenByDescending(p => p.NetworkFee)

This would put HighPriority on top.

Reply:
As discussed, best ordering is the one proposed by Erik:
.OrderByDescending(p => p.NetworkFee / p.Size)
.ThenByDescending(p => p.NetworkFee)
.ThenByDescending(p => InHighPriorityList(p))

Reply:
I’ll need to make a change like this in my PR for improving large MemoryPool behavior, or it won’t necessarily reverify claim transactions ahead of other free transactions.